save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -217,9 +232,39 @@ external interface ProjectDemoMenuProps : Props {
      */
     var organizationName: String
 
+    /**
+     * User project role
+     */
+    var projectRole: Role
```

Comment:
I guess we can be replaced by a single role:
```kotlin
val userRole: Role = when {
    props.globalRole.isSuperAdmin() -> Role.SUPER_ADMIN
    props.organizationRole.isHigherOrEqualThan(Role.OWNER) -> props.organizationRole
    else -> props.projectRole
}
```

Reply:
I think it is possible, but such back roles will cause a lot of questions in subsequent use.
I'm thinking of redoing it soon with backend requests in the future.

Reply:
Take into consideration the fact that the server is in China so requests might take more time then just `when` that Nariman suggested